### DISCUSSION TITLE
Deeplx 今天突然不能使用了

### DISCUSSION THREAD

**Discussion by peter-RenPer** [OwO-Network/DLX#73] [Q&A]
好像是服务器错误
![Uploading image.png…]()

**Comment by missuo** [comment]
> 好像是服务器错误
> 
> ![Uploading image.png…]()
> 
> 

看不到图片

**Reply by peter-RenPer** [reply]
![Uploading iShot_2023-11-24_11.57.35.png…]()

**Reply by peter-RenPer** [reply]
该字符串与预期的模式不匹配。点此测试服务，过墙之后就对了，应该是被墙了
